Project.toml

```diff
@@ -62,7 +62,7 @@ PrettyTables = "1.3, 2.0"
 Reexport = "1"
 StaticArrays = "1"
 YAML = "0.4"
-julia = "~1.10.0"
```

Comment:
I thought the ~ means that we restrict to 1.10.x and don't allow 1.11. This was deliberate. Did I misunderstand something?

Reply:
Everything in this PR is up for discussion, no question. But this CI failure on nightly (which is running 1.12.0-DEV) was caused by the `~`, as best I can tell, and I would prefer to always have the CI passing on `main`.

https://github.com/ACEsuit/ACEpotentials.jl/actions/runs/10500352832/job/29088598779

Reply:
Then turn off nightly

Reply:
I can live with that https://github.com/ACEsuit/ACEpotentials.jl/pull/227/commits/0c1b7cccdeca2e52430560bafef63036ce1a4146